cw-orch/src/daemon/networks/archway.rs

```diff
@@ -17,3 +20,29 @@ pub const CONSTANTINE_1: ChainInfo = ChainInfo {
     lcd_url: Some("https://api.constantine-1.archway.tech"),
     fcd_url: None,
 };
+
+/// Archway Docs: <https://docs.archway.io/resources/networks>
+/// Parameters: <https://testnet.mintscan.io/archway-testnet/parameters>
+pub const CONSTANTINE_3: ChainInfo = ChainInfo {
+    kind: ChainKind::Testnet,
+    chain_id: "constantine-3",
+    gas_denom: "aconst",
+    gas_price: 1000000000000.0,
+    grpc_urls: &["https://grpc.constantine.archway.tech:443"],
+    network_info: ARCHWAY_NETWORK,
+    lcd_url: Some("https://api.constantine.archway.tech"),
+    fcd_url: None,
+};
+
+/// Archway Docs: <https://docs.archway.io/resources/networks>
+/// Parameters <https://www.mintscan.io/archway/parameters>
+pub const ARCHWAY_1: ChainInfo = ChainInfo {
+    kind: ChainKind::Mainnet,
+    chain_id: "archway-1",
+    gas_denom: "aarch",
+    gas_price: 1000000000000.0,
```

Comment:
Interesting, it's 900000000000 rn: https://api.mainnet.archway.io/archway/rewards/v1/estimate_tx_fees How often do they update it? Hopefully we can have it as constant

Reply:
That's the minimum gas price, according to the chain-registry: https://github.com/cosmos/chain-registry/blob/master/archway/chain.json#L20-L23